PUBLISHING-AWS.md

```diff
@@ -133,13 +133,13 @@ cargo make -e SSM_TARGET=latest promote-ssm
 This will copy the fully versioned parameter from earlier, something like:
 
 ```
-/your/prefix/here/aws-k8s-1.24/x86_64/1.11.0-b530f30/image_id
+/your/prefix/here/aws-k8s-1.32/x86_64/1.11.0-b530f30/image_id
```

Comment:
nit: we should update the version to a meaningful one, there is no aws-k8s-1.32 for Bottlerocket 1.11.0.